src/feditest/protocols/webfinger/traffic.py

```diff
@@ -34,10 +34,7 @@ def __init__(self, jrd: 'ClaimedJrd', msg: str):
 
 
         def __str__(self):
-            if self._msg:
-                return f'{ self.__class__.__name__ }: { self._msg }'
-            else:
-                return self.__class__.__name__
+            return { self._msg } or self.__class__.__name__
```

Comment:
Returning `{ self._msg }` ( a Python `set`) from `__str__` causes subsequent exception formatting to fail (e.g., in `wf_error`).

```
__str__ returned non-string (type set)
```

Reply:
Having some difficulty seeing where that would come from. `self._msg` is declared as a `str` and all the constructor invocations I can see look fine. Have some more context?

Reply:
Duh. Rookie mistake.